client/src/locales/en-US.json

```diff
@@ -711,9 +711,9 @@
         "errors": {
             "saveDeviceFailed": "Failed to save the device",
             "sendUserInfoFailed": "Failed to send information to the host, please restart the process.",
-            "noneCodeSelected": "If you didn't see the correct code, it could be a security concern. Please restart the process.",
+            "noneCodeSelected": "If you did not see the correct code, it could be a security concern. Please restart the process.",
```

Comment:
```suggestion
            "noneCodeSelected": "If you did not see the correct code, this could be a sign of a security issue during the process. Please restart the onboarding process.",
```

Reply:
Same as before.